### PR TITLE
Add yum to rawhide and don't remove ci user if it exist

### PR DESCRIPTION
**Pull request by pbrezina**

### CODE REVIEW

**Comment by pbrezina** [comment]
Multihost tests and rawhide fails because yum is not available (we removed it together with dnf5) so we add it back now.

The second patch is per Jakub's request as he's adding idempotency requirement to provision VMs as well with the same playbook.

File: src/ansible/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@
     state: absent
     remove: yes
     force: yes
-  when: regular_user_check.stdout | length > 0
+  when: regular_user_check.stdout | length > 0 and regular_user_check.stdout != user.regular.name
```
Comment (jakub-vavra-cz) [inline src/ansible/roles/common/tasks/main.yml:78]: Without checking I would be vary if there is not <LF> characters in  regular_user_check.stdout

Comment (pbrezina) [reply src/ansible/roles/common/tasks/main.yml:78]: Works in my tests.

**Review by jakub-vavra-cz — APPROVED** [review]
LGTM.

**Review by ikerexxe — APPROVED** [review]
LGTM. Thanks for the patch!

**Pull request merged**